src/utils/localStorage.ts

```diff
@@ -0,0 +1,25 @@
+export const getFromLocalStorage = <T>(key: string): T | null => {
```

Comment:
We can also add a helper hook for working with the localStorage. What do you think?
Here is code of this hook:
``` typescript
import { useState, useEffect } from "react";

export const useStorageState = <S = undefined>(key: string, initial: S) => {
  const [value, setValue] = useState<S>(() => {
    const saved = localStorage.getItem(key);
    if (saved) {
      return JSON.parse(saved);
    }
    return initial;
  });

  useEffect(() => {
    localStorage.setItem(key, JSON.stringify(value));
  }, [value]);

  return [value, setValue] as [typeof value, typeof setValue];
};
```

Reply:
Sure!